product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Remediating AWS control findings"
+
+---
+
+This page describes the control findings in your AWS Security Hub and suggests potential remediation steps for your BigAnimal resources. 
```

Comment:
I would remove the term "control" which is of very widespread use (we tend to use it a lot internally, I may have used it myself when discussing this), but difficult interpretation. Moreover it is not used at all in the security hub documentation, which only talks about "findings".

Suggested rewording: "This page describes findings in your AWS Security Hub...".

Whether we want to add also "that may relate to BigAnimal cloud resources" is an interesting point, I think we should if that's not clear/implied by other pieces of documentation. And especially we should be consistent with what is discussed on the Azure side.

Reply:
Incorporated your feedback.